### PR TITLE
5. Add Reports tab

### PR DESCRIPTION
**Pull request by yaroslavmo**
Add reports tab with service reports table

### CODE REVIEW

File: app/admin_components/adf-reports/views/df-manage-service-reports.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                            <li data-ng-click="exportServiceReports('json')">
+                                <a>JSON</a>
+                            </li>
+                            <li data-ng-click="exportServiceReports('xml')">
+                                <a>XML</a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>-->
```
Comment (OlegLazaryev) [inline app/admin_components/adf-reports/views/df-manage-service-reports.html]: Let's remove the commented code.

File: app/admin_components/adf-reports/views/df-manage-service-reports.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                                            <label>
+                                                <input id="reports_table_row_checkbox_{{obj.record.id}}" type="checkbox" data-ng-checked="obj.__dfUI.selected" data-ng-click="setSelected(obj)" />
+                                                <div class="btn-group btn-group-xs">
+                                                    <button id="reports_table_row_delete_{{obj.record.id}}" type="button" class="btn btn-default" data-ng-click="deleteServiceReport(obj)" title="Delete report"><i class="fa fa-trash"></i></button>
+                                                </div>
+                                            </label>
+                                        </div>
+                                    </td>-->
+                                    <!-- Auto build table based on fieldorderobj in directive -->
+                                    <!-- <td data-ng-click="editAdmin(obj.record)" data-ng-repeat="value in obj.record | orderAndShowValue : fields track by $index">{{value}}</td>-->
```
Comment (OlegLazaryev) [inline app/admin_components/adf-reports/views/df-manage-service-reports.html]: Let's remove commented code

File: app/scripts/controllers/main.js [hunk 3]
```diff
@@ -196,15 +191,99 @@ angular.module('dreamfactoryApp')
...
+                }
+            };
+
+            function splitSchemaDataTab(accessibleTabs, schemaDataIndex) {
+                var schemaDataArr = accessibleTabs[schemaDataIndex].split('/');
+                accessibleTabs.splice(schemaDataIndex, 1, schemaDataArr[0], schemaDataArr[1]);
+                return accessibleTabs;
+            }
+
+            function getConfigTabInsertIndex(accessibleLinks, tabsLinks) {
```
Comment (OlegLazaryev) [inline app/scripts/controllers/main.js]: Let's discuss this function.

File: app/admin_components/adf-reports/views/df-manage-service-reports.html (hunk 1 above)
Comment (yaroslavmo) [reply app/admin_components/adf-reports/views/df-manage-service-reports.html]: Done

File: app/scripts/controllers/main.js (hunk 3 above)
Comment (yaroslavmo) [reply app/scripts/controllers/main.js]: Ok. But let's go to this [PR](https://github.com/dreamfactorysoftware/df-admin-app/pull/113), as this function concerns the RA feature.

**Pull request closed without merging**